web/src/composables/useLogs.ts

```
@@ -553,7 +555,10 @@ const useLogs = () => {
 
   const updateUrlQueryParams = () => {
     const query = generateURLQuery(false);
-    if(query.hasOwnProperty("type") && query.type == "search_history_re_apply"){
+    if (
+      query.hasOwnProperty("type") &&
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)